app/login/page.js

```diff
@@ -3,15 +3,21 @@ import Image from 'next/image'
 
 import logo from '@/public/main-logo.png'
 import { useAuth } from '../context/AuthContext'
-import { useRouter } from 'next/navigation'
+import { useState } from 'react'
 
 const LoginPage = () => {
-  const { handleGoogleLogin } = useAuth()
-  const router = useRouter()
+  const { handleGoogleLogin, showLoginError, setShowLoginError } = useAuth()
+
+  // Handle Google auth
+  const handleGoogleSignin = () => {
```

Comment:
Why is handleGoogleLogin() inside the constant handleGoogleSignin? This variable is not being used.